src/scss/components/switch/mixins/_hxSwitchFacade.scss

```diff
@@ -0,0 +1,61 @@
+//
+// ===== Mixins for <hx-checkbox> facade states =====
+//
+
+// ========== PRISTINE ==========
+@mixin __checkboxFacade($pseudo-state: null) {
```

Comment:
Please change to:

```scss
@mixin __switchFacade($pseudo-state: null) {
```

We will have to confirm if these these are `<hx-checkbox>` or `<hx-switch>` styles, and also confirm that they are used.

Reply:
As facade mixins are not used , i removed this file